var/spack/repos/builtin/packages/flux-pmix/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack import *
+
+
+class FluxPmix(AutotoolsPackage):
+    """ A flux shell plugin to bootstrap openmpi v5+ """
+
+    homepage = "https://github.com/flux-framework/flux-pmix"
+    url      = "https://github.com/flux-framework/flux-pmix/releases/download/v0.1.0/flux-pmix-0.1.0.tar.gz"
+    git      = "https://github.com/flux-framework/flux-pmix.git"
+
+    maintainers = ['grondo']
+
+    version('main', branch='main')
+    version('0.2.0', sha256='d09f1fe6ffe54f83be4677e1e727640521d8110090515d94013eba0f58216934')
+
+    depends_on('flux-core@0.30.0:')
+    depends_on('pmix@v4.1.0:')
+    depends_on('openmpi')
+
+    #  Some older pkg-config versions cause errors with ompi.pc file,
+    #   so require latest version from spack:
+    depends_on('pkg-config', type='build')
+
+    def autoreconf(self, spec, prefix):
+        if not os.path.exists('configure'):
+            # Bootstrap with autotools
+            bash = which('bash')
+            bash('./autogen.sh')
```

Comment:
This suggests we might be missing a:
```
depends_on('autoconf', type='build')
depends_on('automake', type='build')
...
```
Can you double check that?

Reply:
Yes, `flux-pmix` requires `autoconf`, `automake` and `libtool` at build time. I will add those. Thanks!

Reply:
I've added these packages as build dependencies and pushed the result.